### PR TITLE
APIS-6879: Subproject for integration tests

### PR DESCRIPTION
**Pull request by johnsgp**

### CODE REVIEW

File: project/ScoverageSettings.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import scoverage.ScoverageKeys._
+import scoverage.ScoverageKeys.*
```
Comment (AndySpaven) [inline project/ScoverageSettings.scala]: Scala 3 syntax already ?  Bit keen

Comment (johnsgp) [reply project/ScoverageSettings.scala]: It seems to work and it removes a warning from IntelliJ. I can revert if you prefer but I've done it elsewhere too.

Comment (johnsgp) [reply project/ScoverageSettings.scala]: I've set it back to `_`

Comment (AndySpaven) [reply project/ScoverageSettings.scala]: IntelliJ is getting ahead of itself... probably thinks we should be using Scala 3 after it's been out for 5 years :-)

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**